components/RecentlyPlayedDrawer.tsx

```diff
@@ -215,14 +379,14 @@ export default function RecentlyPlayedDrawer({}: RecentlyPlayedDrawerProps) {
     ),
   }));
 
-  const renderSong = (song: ProcessedSong, index: number) => {
+  const renderSong = (song: ProcessedSong, key: string) => {
```

Comment:
future - would be better as its own component for organization/to let React bail sooner (will require moving `key` up a level). but may require refactoring/prop drilling to get everything wired neatly